FEATURE_MATRIX.md

```diff
@@ -38,21 +45,21 @@
 ## Planning
 | | C# | Python | Notes |
 |---|---|---|---|
-| Plan | ✅ | ❌ | |
-| SequentialPlanner | ✅ | ❌ | |
+| Plan                              | ✅ | ❌ | |
+| SequentialPlanner                 | ✅ | ❌ | |
 
 ## Connectors and Skill Libraries
 | | C# | Python | Notes |
 |---|---|---|---|
 | Qdrant (Memory)                   | ✅ | ❌ | Vector optimized |
-| ChromaDb (Memory)                 | ❌ | 🔄 | |
+| ChromaDb (Memory)                 | ❌ | ✅ | |
 | Milvus (Memory)                   | ❌ | ❌ | Vector optimized |
 | Pinecone (Memory)                 | ✅ | ❌ | Vector optimized |
 | Weaviate (Memory)                 | ❌ | ❌ | Vector optimized |
 | CosmosDB (Memory)                 | ✅ | ❌ | CosmosDB is not optimized for vector storage |
 | Sqlite (Memory)                   | ✅ | ❌ | Sqlite is not optimized for vector storage |
 | Postgres (Memory)                 | ✅ | ❌ | Vector optimized (required the [pgvector](https://github.com/pgvector/pgvector) extension) |
-| Azure Cognitive Search            | ❌ | ❌ | |
+| Azure Cognitive Search            | 🔄 | ❌ | Azure Search backed by vector search coming soon |
```

Comment:
I'd mark as enabled for now and allow any PRs that are doing changes in this space to update as appropriate.